tests/test_pipeline_orchestrator.py

```
@@ -38,8 +37,8 @@
     from moml.pipeline import PFASPipelineOrchestrator
 
     PIPELINE_AVAILABLE = True
-except ImportError as e:
-    logger.warning(f"Pipeline module not available: {e}")
+except ImportError:
+    logger.warning(f"Pipeline module not available")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)